source/configure-connection.txt

```diff
@@ -68,37 +74,43 @@ This nested dictionary has the following keys:
      - | Your connection URI. For localhost connections, this key is optional. 
        | For SRV connections, you must include a scheme prefix (``mongodb+srv://``).
        |
-       | If connecting to a replica set or sharded cluster with multiple hosts, specify 
-       | each host separated by a comma.
+       | To specify more than one host, include all hostnames in one string. Use
+         a comma to separate each hostname.
+       | **Example:** ``"HOST": "mongodb://mongos0.example.com:27017,mongos1.example.com:27017"``
 
    * - **NAME**
      - The database you want to use.
 
    * - **USER**
-     - The username for authenticating to the database.
+     - The username for authenticating to the database, if your connection
+       requires authentication.
 
    * - **PASSWORD**
-     - The password for your database user.
+     - The password for your database user, if your connection requires authentication.
 
    * - **PORT**
-     - | The port number on which the database server is listening. 
+     - | The port number on which the database server is listening. The default
+         port is ``27017``.
        | For MongoDB Atlas connections, this key is optional.
 
    * - **OPTIONS**
-     - A dictionary of additional connection options for the database. This key is optional.
+     - | A dictionary of additional connection options for the database. This key is optional.
+       | To see a full list of connection options that you can set in the ``OPTIONS`` key,
+         see the optional parameters for `MongoClient <https://pymongo.readthedocs.io/en/4.10.1/api/pymongo/mongo_client.html#pymongo.mongo_client.MongoClient>`__
+         in the PyMongo API documentation.
 
 .. _django-manual-config-example:
 
 Example 
 ```````
 
-This example specifies the ``DATABASES`` variable to connect
-to a MongoDB deployment with the following configuration:
+In this example, the ``DATABASES`` variable specifies the
+following connection configuration:
 
-- Connects to the ``my_database`` database
+- Sets the database to ``my_database``
 - Provides authentication information for a database user
   whose username is ``my_user`` and password is ``my_password``
-- Uses the default MongoDB port (``27017``)
+- Specifies the default MongoDB port (``27017``)
 - Sets the ``retryWrites`` connection option to ``true``,
```

Comment:
i: there's a double verb here--'specifies' in the paragraph and then 'connects'/'sets' in the list items. i can't add a suggestion in GH, but maybe something like this:

In this example, the ``DATABASES`` variable performs the
following steps:

Reply:
changed to just "...performs the following actions"